docs/src/pages/_document.page.tsx

```diff
@@ -28,7 +28,7 @@ const getCSPContent = (context: Readonly<DocumentProps>) => {
       img-src 'self' blob: ${ANALYTICS_CSP.all.img.join(' ')};
       connect-src 'self' *.shortbread.aws.dev ${ANALYTICS_CSP.all.connect.join(
         ' '
-      )} https://*.algolia.net https://*.algolianet.com https://cdn.jsdelivr.net https://tfhub.dev https://storage.googleapis.com;
+      )} https://*.algolia.net https://*.algolianet.com https://cdn.jsdelivr.net https://tfhub.dev https://storage.googleapis.com https://cdn.liveness.rekognition.amazonaws.com;
```

Comment:
Is this change related to the example?

Reply:
Not exactly related but is a needed change, will make it in a separate pr